.github/workflows/cron_jerigon_zero_testing.yml

```diff
@@ -0,0 +1,106 @@
+name: Jerigon Zero Testing
+
+on:
+  # TODO - Change this before merge
+  # # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 12 * * SUN"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  jerigon_zero_testing:
+    name: Jerigon Zero Testing - Integration and Benchmarking
+    runs-on: zero-reg
+    concurrency:
+      group: jerigon_zero_testing
+      cancel-in-progress: true
+    steps:
+      - name: Checkout zk_evm code
+        uses: actions/checkout@v4
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Set up Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
```

Comment:
nit: these have been refactored - please use the refactored versions in-repo

Reply:
can you please point it out the refactored version. couldn't find

Reply:
```yaml
    steps:
      - uses: actions/checkout@v4
      - uses: ./.github/actions/rust
 ```
https://github.com/0xPolygonZero/zk_evm/blob/dd17ff3d61f03c8eef73c41b2931575fb1de8082/.github/workflows/lint.yml#L23-L27